Universal model for SCI lesions (#87)
ivadomed/model_seg_sci: README.md

[![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf)

This repository contains the code for deep learning-based segmentation of the spinal cord and intramedually lesions in spinal cord injury (SCI). The code is based on the [nnUNetv2 framework](https://github.com/MIC-DKFZ/nnUNet).

## Model Overview

> The model was trained on raw T2-weighted images of SCI patients from seven sites comprising traumatic (acute preoperative, intermediate, chronic) and non-traumatic (degenerative cervical myelopathy, DCM) SCI lesions. The data included images with heterogenous resolutions (both axial and sagittal) and scanner strenghts (1T/1.5T/3T). To ensure uniformity across sites, all images were initially re-oriented to RPI. Given an input image, the model is able to segment *both* the lesion and the spinal cord. 

Reviewer: ```suggestion
The model was trained on raw T2-weighted images of SCI patients from seven sites comprising traumatic (acute preoperative, intermediate, chronic) and non-traumatic (ischemic SCI and degenerative cervical myelopathy, DCM) SCI lesions. The data included images with heterogenous resolutions (both axial and sagittal) and scanner strengths (1T/1.5T/3T). To ensure uniformity across sites, all images were initially re-oriented to RPI. Given an input image, the model is able to segment *both* the lesion and the spinal cord. 
```

The outputs will be saved in the same directory as the input image, with the suffix `_lesion_seg.nii.gz` for the lesion 
and `_sc_seg.nii.gz` for the spinal cord.

### Automatic measurements of midsagittal tissue bridges

> This new functionality is available via `sct_analyze_lesion`. The function computes the midsagittal tissue bridges and outputs the ventral and dorsal tissue bridges. 

Reviewer: ```suggestion
This new functionality is available via [SCT](https://github.com/spinalcordtoolbox/spinalcordtoolbox)'s `sct_analyze_lesion`. The function computes the midsagittal tissue bridges and outputs the ventral and dorsal tissue bridges. 
```

```bash

> sct_analyze_lesion -h

Reviewer: ```suggestion
sct_analyze_lesion -m <SUBJECT>_lesion_seg.nii.gz -s <SUBJECT>_sc_seg.nii.gz
```

Author: sure! accepted
